package.json

```
@@ -1,9 +1,15 @@
 {
   "name": "drand-client",
-  "version": "1.0.0-pre.0",
+  "version": "1.0.0-pre.1",
   "description": "A client to the drand randomness beacon network.",
   "type": "module",
-  "main": "lib/drand.ts",
+  "main": "build/index.js",
```

```suggestion
  "main": "build/index.js",
  "types": "build/index.d.ts",
```

I guess you need this too